src/SuperMassive/Helpers/SonarHelper.cs

```diff
@@ -0,0 +1,13 @@
+namespace SuperMassive
+{
+    using System;
+
+    public class SonarHelper
```

Comment:
Codacy found an issue: [Add a 'protected' constructor or the 'static' keyword to the class declaration.](https://app.codacy.com/manual/PulsarBlow/SuperMassive/pullRequest?prid=6193423)